spm_test.py

```diff
@@ -0,0 +1,93 @@
+# LiC6 -> C6 + Li+ + e- (reaction at the anode)
+# I do not track the movement of Li through the anode, only the rate of creation of Li+ at the surface
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+from scipy.integrate import solve_ivp
+from spm_functions import Half_Cell_Eqlib_Potential,Butler_Volmer,Species,Participant,Half_Cell
+
+# Held Constant
+Vol_Frac = 0.5 # Fraction of the Anode that is ocupied by Lithium [m^3/m^3]
+MV = 13*10**-6 # Molar Volume of Lithium [m^3/mol]
+nu_Li_plus = 1 # stoichiometric coefficient of Lithium [mol_Li+/mol_rxn]
+#C_Li_plus = 1 # Concentration of Li+ in the Electrolyte [mol/L] (for use later on when U is not constant)
+i_o = 120 # Exchange Current Density [A/m^2] 
+C_dl = 6*10**3 # Double Layer Capacitance [F/m^2]
+T = 273.15 + 25 # standard temperature [K]
+U = 1 # Open circuit (equalibrium) potential [V]
+i_ext = -2 # external current [A]
+F = 96485.34 #Faraday's number [C/mol_electron]
+R = 8.3145 #Universal gas constant [J/mol-K]
+n = 1 # number of electrons [mol_electrons/mol_rxn]
+Beta = 0.5 # [-] Beta = (1 - Beta) in this case 
+BnF_RT = Beta*n*F/R/T # combine constants for simplicity
```

Comment:
Eventually, you will want a `BnF_RT_a` and `BnF_RT_c = (1-Beta)*n*F/R/T` for the anodic and cathodic directions.  They are the same only so long at `Beta = 0.5`,  but since Beta can be varied here, would be wise to implement this.